cfme-performance/tests/workloads/test_capacity_and_utilization.py

```diff
@@ -34,17 +36,18 @@ def test_workload_capacity_and_utilization(request, scenario):
         'scenario': scenario}
     monitor_thread = SmemMemoryMonitor(SSHClient(), scenario_data)
 
-    def cleanup_workload(scenario, from_ts):
+    def cleanup_workload(scenario, from_ts, quantifiers, scenario_data):
         starttime = time.time()
         to_ts = int(starttime * 1000)
         g_url = get_scenario_dashboard_url(scenario, from_ts, to_ts)
         logger.debug('Started cleaning up monitoring thread.')
         monitor_thread.grafana_url = g_url
         monitor_thread.signal = False
         monitor_thread.join()
+        add_workload_quantifiers(quantifiers, scenario_data)
```

Comment:
I am a bit confused here.  Can the quantifiers be "sent" to the report generator in another way?  It seems redundant that scenario_data is passed into the the monitor_thread and into the finalizer.  I believe we need a more sustainable report generation/separation in the future however that will have to be worked on separately.  I think we can accomplish the same thing in a bit less code and in a "clearer" fashion.  Thoughts?